code/__DEFINES/antagonists.dm

```diff
@@ -0,0 +1,37 @@
+//The file that the threat datum will read the probabilities
+#define ANTAG_PROBABILITIES_FILE	'config/antag_probabilities.txt'
+
+#define ANTAG_DATUM_PATH			"/datum/antagonist/"
```

Comment:
This is mainly for the file reading(so you don't have to type the entire string in the config file), which will read as a string, and I was not sure if you could do `path + string`.